Soccar [Client]/src/nl/soccar/ui/DisplayConstants.java

```diff
@@ -8,29 +8,32 @@
 public final class DisplayConstants {
 
     /**
-     * Screen properties
+     * Screen constants in pixels
      */
-    public static final int SCREEN_WIDTH = DisplayConstants.MAP_WIDTH + DisplayConstants.FIELD_MARGIN;
-    public static final int SCREEN_HEIGHT = DisplayConstants.MAP_HEIGHT + DisplayConstants.FIELD_MARGIN;
+    public static final float SCREEN_WIDTH = 1200;
+    public static final float SCREEN_HEIGHT = 800;

```

Comment:
TODO: Replace with the viewport resolution of the actual screen of the device in use.